webpack.config.js

```diff
@@ -10,6 +10,15 @@ const { CleanWebpackPlugin } = require('clean-webpack-plugin');
 const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
 const FileManagerPlugin = require('filemanager-webpack-plugin-fixed');
 
+let runningChunkId = 0;
+const runningChunkMap = {};
+
+const getSafeChunkFilename = (extension) => (a) => {
```

Comment:
Ok, so here's the happy balance between being resilient and being sloppy.  Starting with an id of 1, we incrementally name each unknown chunk.

Then, for chunks 1 to 10 (we only have 2 unknown now), we try to add them to the exported gallery (cf line 329).  This means that the code should never need to be updated again, but you'll see a bit of red as it attempts to get chunks 3 through 8 as it does its gallery export.

Reply:
Webpack config in general is not my strong suit, but I looked into this again not what you brought it up and I think I may have found the solution I wanted. I approached it from a different angle this time. Every named chunk had a name because I had given them names during import, so I looked through the imports if there was anything to improve, but there was still those few that I just couldn't name without causing other issues...